lib/adf_diag.py

```diff
@@ -946,10 +946,16 @@ def derive_variables(self,vars_to_derive=None,ts_dir=None):
             if var == "PRECT":
                  # PRECT can be found by simply adding PRECL and PRECC
                  # grab file names for the PRECL and PRECC files from the case ts directory
-                 if len(glob.glob(os.path.join(ts_dir,"*PRECC*"))) and len(glob.glob(os.path.join(ts_dir,"*PRECL*"))):
+#                 if len(glob.glob(os.path.join(ts_dir,"*PRECC*"))) and len(glob.glob(os.path.join(ts_dir,"*PRECL*"))):
+#                     constit_files=sorted(glob.glob(os.path.join(ts_dir,"*PREC*")))
+#                 else:
+#                     ermsg = "PRECC and PRECL were not both present; PRECT cannot be calculated."
+#                     raise FileNotFoundError(ermsg)
```

Comment:
Is there a reason you commented this out instead of just deleting it?

Reply:
Done.